docs/rbac/index.rst

```diff
@@ -0,0 +1,26 @@
+.. _rbac-index:
+
+Role-based Access Control
+=========================
+
+Role-based access control (RBAC) **restricts** access to entities based on a user's role within an
+organization. A role consists of one or more permissions. Users having a proper set of roles can
+view, modify, or delete content hosted on different endpoints.
+
```

Comment:
maybe worth adding a section called 'Content access'

Reply:
can you also add information about the queryset scoping?

Reply:
Not sure how the information about the queryset scoping will be useful to users. It tells plugin-writers that Pulp restricts access to objects based on users' permissions (which is generally known): https://github.com/pulp/pulpcore/blob/8ab3fdfba539b81fa25b67876dde7327992a6ce4/docs/plugins/plugin-writer/concepts/rbac/queryset_scoping.rst